lib/rules/no-control-regex.js

```diff
@@ -51,29 +51,47 @@ module.exports = {
             return null;
         }
 
+
+        const controlChar = /[\x00-\x1f]/g; // eslint-disable-line no-control-regex
+        const multipleSlashes = /\\+/g;
+        const multipleSlashesAtEnd = /\\+$/g;
```

Comment:
Sorry for being dense, but I guess I still don't understand. From what I can tell, the regexes are matching one regex or more (`\\` is one backslash since `\` escapes the next character). Are we asserting that there is another backslash before/after the pattern match location to get our "2 or more" result? Or am I missing something?